user-guide/Standard_Apps/SRM/Interacting_with_SRM_using_code/SRM_silent_actions/SRM_update_timing_booking_silently.md

```diff
@@ -49,3 +49,8 @@ public class Script
  }
 }
 ```
+
+> [!NOTE]
+> Since SRM Framework 1.2.37/2.0.2 <!-- RN40481 --> it's possible to finish a permanent booking while defining post-roll. To do so, the timing of that booking should be changed, by changing the end date to a time in the near future and setting the appropriate post-roll time.
+> Finishing a permanent booking will not support defining any post-roll.
```

Comment:
This sentence isn't entirely clear to me, as it seems to contradict what was said just above. Do you mean that you can only define a post-roll time if you finish a permanent booking by defining an end time for it, but not if you finish the booking in a different way?